AWSLambdaTests/AWSLambdaInvokerTests.m

```diff
@@ -238,6 +238,8 @@ - (void)testInvokeFunctionError {
 
 - (void)testInvokeWithVersion {
```

Comment:
There is another `testInvokeWithVersion` in AWSLambdaTests.m

Reply:
fixed